components/shared/FooterClient.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { FooterQuery } from '../../tina/__generated__/types';
+import {
+  FaYoutube,
+  FaLinkedin,
+  FaFacebook,
+  FaTwitter,
+  FaInstagram,
+  FaTiktok,
+  FaGithub,
+} from 'react-icons/fa';
+
+interface FooterClientProps {
+  results: FooterQuery | null;
+}
+
+const iconMap: { [key: string]: JSX.Element } = {
+  FaYouTube: <FaYoutube />,
+  FaLinkedIn: <FaLinkedin />,
+  FaFacebook: <FaFacebook />,
+  FaTwitter: <FaTwitter />,
+  FaInstagram: <FaInstagram />,
+  FaTiktok: <FaTiktok />,
+  FaGithub: <FaGithub />,
+};
+
+export default function FooterClient({ results }: FooterClientProps) {
+  const [isVisible, setIsVisible] = useState(false);
+
+  useEffect(() => {
+    setIsVisible(true);
+  }, []);
+
+  const footerItems = results?.footer?.footer;
+  const footerTitle = results?.footer?.footerTitle;
+  //Removed #00000 fallback color and replaced with undefined so line 46 doesnt potentially get a null value
+  const footerColour = results?.footer?.footerColour || undefined; 
```

Comment:
As per our conversation, a fallback is better, but should use a predefined tailwind color

Reply:
done